### PR TITLE
Remove Rabbit and Spring integration xml

### PR DESCRIPTION
**Pull request by warrenbailey**
# What and why?
Remove the last bits of the spring integration xml files and the connection to rabbit as this is all done via pubsub now.

# How to test?

Deploy to an env and run acceptance tests

# Trello
https://trello.com/c/cPLz5WEs/767-remove-final-case-service-rabbit-spring-integration

### CODE REVIEW

**Review by insacuri — CHANGES_REQUESTED** [review]
One super small comment, but looks good

File: _infra/helm/case/Chart.yaml [hunk 1]
```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 11.1.53
+version: 11.1.54
```
[the remark this answers is not in the document]
Comment (warrenbailey) [reply _infra/helm/case/Chart.yaml:17]: good shout

**Review by insacuri — APPROVED** [review]

**Pull request merged**